app/webpacker/styles/application.scss

```diff
@@ -56,7 +56,7 @@ $govuk-image-url-function: frontend-image-url;
 
 .govuk-width-container__wide {
   @media (min-width: 1110px) {
-    min-width: 1110px;
+    min-width: 1170px;
```

Comment:
I've slightly widened the layout so that long UUID do not wrap and summary lists displayed side by side keeps their respective rows aligned